secboot/secboot_tpm.go

```diff
@@ -278,6 +280,28 @@ func UnlockVolumeIfEncrypted(disk disks.Disk, name string, encryptionKeyDir stri
 	return filepath.Join("/dev/disk/by-partuuid", partUUID), false, nil
 }
 
+// UnlockVolumeUsingSealedKeyIfEncrypted unlocks an existing volume using the
```

Comment:
This should be UnlockEncryptedVolumeUsingKey.

Reply:
Thanks for spotting, I'll do quick followup

Reply:
Addressed in #9564